meson.build

```diff
@@ -0,0 +1,28 @@
+project('diffuse',
+    version: '0.6.0',
+    meson_version: '>= 0.50',
+    license: 'GPL2+')
```

Comment:
Very nitty, but usually one should follow SPDX spec, so GPL-2.0-or-later might be more fitting here

Reply:
Hmmm I took that from the other projects and I'm reading on [Wikipedia](https://en.wikipedia.org/wiki/Software_Package_Data_Exchange) it's a valid syntax:
> There is also a "+" operator, when applied to a license, means that future versions of the license apply as well.

But digging in the SPDX specifications, it was valid in the [version 1.2](https://spdx.dev/wp-content/uploads/sites/41/2017/12/spdx-1_2.pdf) (see section _5.1 Identifier Assigned_ and _Appendix I: Standard License List_), but it is apparently no longer a valid syntax starting v2.0 ([link to v2.2 because it's simpler than in a PDF](https://spdx.github.io/spdx-spec/appendix-I-SPDX-license-list/#i3-deprecated-licenses)).

GPL-2.0-or-later it is then!

(I'll update the Wikipedia page)